mgradm/shared/templates/pgsqlVersionUpgradeScriptTemplate.go

```diff
@@ -24,6 +24,8 @@ echo "Testing presence of postgresql$OLD_VERSION..."
 test -d /usr/lib/postgresql$OLD_VERSION/bin
 
 echo "Create a backup at /var/lib/pgsql/data-backup..."
+SUFFIX=$(date "+%Y%m%d_%H%M%S")
+test -d "/var/lib/pgsql/data-backup" && mv "/var/lib/pgsql/data-backup" "/var/lib/pgsql/data-backup-${SUFFIX}"
```

Comment:
Wouldn't this be a bit more readable?

```suggestion
test -d "/var/lib/pgsql/data-backup" && mv "/var/lib/pgsql/data-backup" "/var/lib/pgsql/data-backup-$(date '+%Y%m%d_%H%M%S')"
```